templates/vue/src/components/TapestryMap.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+
+  <div style="height: 800px; width: 100%">
+    <div style="height: 150px overflow: auto;">
+      <p>First marker is placed at {{ withPopup.lat }}, {{ withPopup.lng }}</p>
+      <p> The bounds are: {{ mapBounds }} </p>
+      <p> Settings are: {{this.settings}} </p>
+    </div>
+    <l-map
+      :zoom="zoom"
+      :center="center"
+      :options="mapOptions"
+      :max-bounds="mapBounds"
+      :bounds='setBounds'
+      style="height: 80%"
+      @update:center="centerUpdate"
+      @update:zoom="zoomUpdate"
+     
+    >
+      <l-tile-layer
+        :url="url"
+        :attribution="attribution"
+
+      />
+    </l-map>
+  </div>
+</template>
+
+<script>
+import { latLng, latLngBounds } from "leaflet";
```

Comment:
This would usually get caught by the linter, but in our code, we drop the semicolon at the end of each line.